Corona-Warn-App/src/main/res/drawable/ic_lock.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="20dp"
+    android:viewportWidth="16"
+    android:viewportHeight="20">
+  <path
```

Comment:
Is this icon the same for dark and light theme?

Reply:
Yes, all icons are white in both modes